xAPI-Data.md

```diff
@@ -436,7 +437,7 @@ The table below lists the properties of Agent Objects.
 ###### <a name="2.4.2.2.s1"></a>Description
 
 A Group represents a collection of Agents and can be used in most of the same situations an Agent 
-can be used.  There are two types of Groups, anonymous and identified.
+can be used.  There are two types of Groups, Anonymous Groups and Identified Groups.
```

Comment:
Should this be:

There are two types of Groups: Anonymous Groups and Identified Groups.

(colon instead of comma)